readme.md

```diff
@@ -71,6 +71,7 @@ Feel free to play around with the code and fork this repl at [https://repl.it/@v
 - [swiff](https://github.com/simple-integrated-marketing/swiff) - Multi-environment command line tools for time-saving web developers.
 - [changelog-view](https://github.com/jdeniau/changelog-view) - Tool view changelog in console.
 - [gomoku-terminal](https://github.com/acrazing/gomoku-terminal) - Play online Gomoku in the terminal.
+- [cfpush](https://cfpush.cloud) - An interactive Cloud Foundry tutorial in your terminal.
```

Comment:
Can you link to the GitHub repo instead?

I would also like to see a screenshot, or even better, an animated GIF of how the CLI looks like.

Reply:
Sure! I updated the link and the project's readme.